lib/automation_extensions/drivers/v4/local.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+module AutomationExtensions
+  module Drivers
+    module V4
+      class Local
+        attr_reader :browser
+        private :browser
```

Comment:
I think you can put the `attr_reader` in the private section to make it private?  Although same thing, I guess?